### DISCUSSION TITLE
"视频预览"功能在"正在关注"页面上不会生效

### DISCUSSION THREAD

**Discussion by FguoDaf** [BewlyBewly/BewlyBewly#656] [Q&A]
首页视频不论是"个性推荐""/"热门视频""还是"排行"都是可以直接预览视频的, 但是正在关注的视频无法启用预览功能, 是bug吗? 期待作者大大修正!

**Comment by hakadao** [comment]
那部分接口沒有提供出 cid 參數導致不能調用預覽功能，所以說面那也強調「如果可以的話」
